en/docs/manage/lifecycle.md

```diff
@@ -1,3 +1,40 @@
 # Lifecycle
 
-This document discsses the lifecycle of a component and how it can be changed. 
\ No newline at end of file
+The API lifecycle is one of the key factors in API management. An API lifecycle has predefined states. These states represent the stages that an API has in the process of starting to develop an API until it's retirement. APIs developed in Choreo have a lifecycle that contains six stages which allows you to identify in which state that the APIs are currently in.
```

Comment:
Suggestion: "The API lifecycle is one of the key factors in API management. An API lifecycle has predefined states. These states represent the different stages of the API from the time you create it until its retirement. An API developed in Choreo has a lifecycle with six states that allow you to identify its current stage."

Reply:
slightly changed and used.

Reply:
The grammatical issues in the original text are still there. What is missing in the suggestion? We can try to incorporate those.

To improve on the suggestion, I think some examples might help as follows:

"The API lifecycle is one of the key factors in API management. An API lifecycle has six predefined states. These states represent the different stages of the API  (for example, whether it is available for subscriptions, outdated, etc.) from the time you create it until its retirement. The API developers update the lifecycle states of their APIs to allow API consumers to identify the stage the API is in at any given time."

Reply:
API lifecycle can have different number of predefined states, Choreo has 6. The last sentence is incorrect actually. 

I am not clear on what the grammatical issue is..

Reply:
Changed slightly. Hope it helps.